pootle/apps/accounts/adapter.py

```diff
@@ -47,3 +48,12 @@ def is_open_for_signup(self, request):
     def add_message(self, request, level, message_template, *args, **kwargs):
         """Silence messages altogether."""
         pass
+
+    def send_confirmation_mail(self, *args, **kwargs):
+        try:
+            super(PootleAccountAdapter, self).send_confirmation_mail(*args,
+                                                                     **kwargs)
+        except Exception:
+            raise RuntimeError(_('Some problem happened when tried to send '
+                                 'the confirmation email. Please try again '
+                                 'later.'))
```

Comment:
After the sign up form is successfully sent (and confirmation email is sent) the user is presented a new window telling them:

```
Your account needs activation.

We have sent an email containing the special link to
YOUR_EMAIL

Please follow that link to continue the account creation.
```